com.unity.template-universal/Packages/manifest.json

```diff
@@ -2,15 +2,10 @@
   "dependencies": {
     "com.unity.2d.sprite": "1.0.0",
     "com.unity.2d.tilemap": "1.0.0",
-    "com.unity.ads": "3.7.5",
+    "com.unity.ads": "3.7.1",
```

Comment:
Don't know if I missed it on the previous PR, do we need this package? (it is not one of the default packages we need to add)
If yes, it should be included in package.json as well and if not - would be great if we can remove it from the manifest.

Reply:
looks like a bad merge from me, thanks for pointing it